mitosheet/mitosheet/mito_backend.py

```diff
@@ -316,6 +316,19 @@ def get_current_kernel_id() -> str:
     file_name: str = os.path.basename(connection_file) 
     return file_name[len('kernel-'):-len('.json')] 
 
+
+def get_mito_frontend_code(kernel_id: str, comm_target_id: str, div_id: str, mito_backend: MitoBackend) -> str:
+    js_code = js_code_from_file.replace('REPLACE_THIS_WITH_DIV_ID', div_id)
+    js_code = js_code.replace('REPLACE_THIS_WITH_KERNEL_ID', kernel_id)
+    js_code = js_code.replace('REPLACE_THIS_WITH_COMM_TARGET_ID', comm_target_id)
+    # NOTE: we need to turn \ into \\ anywhere it exists, as otherwise
```

Comment:
Just update this comment to \\ into \\\\ and finish the sentence, then LGTM. 

I can import the failing file on this branch, but not dev!